hello_world/transactions/test/hello_transaction.test.js

```diff
@@ -0,0 +1,91 @@
+const HelloTransaction = require('../hello_transaction');
+const {TransactionError, utils} = require('@liskhq/lisk-transactions');
+const { when } = require('jest-when');
+
+describe('Hello Transaction', () => {
+    let storeStub;
+    beforeEach(() => {
+        storeStub = {
+            account: {
+                get: jest.fn(),
+                set: jest.fn(),
+            },
+        };
+    });
+
+    test('it should save the hello string in the senders account assets', async () => {
```

Comment:
```suggestion
    it('should save the hello string in the senders account assets', async () => {
```
we usually use `it` phrase here